ssh.go

```diff
@@ -3,8 +3,7 @@ package tmsh
 import (
 	"bytes"
 	"io"
-
```

Comment:
Please insert blank line between standard packages and other packages.
https://github.com/golang/go/wiki/CodeReviewComments#imports